### ISSUE TITLE
Current forced small row height in Flat Data Grid obscures underscores and descender parts of text

### ISSUE THREAD

**Issue by entact-vhorvath** [RandomFractals/vscode-data-table#157]
The current line height obscures important textual information (underscores). This is vital for applications where data contains underscores, e.g. XML based data formats.

Using your examples one can see that the part of the text falling under the baseline is now invisible, see note field, e.g. _untitled_19:

![image](https://github.com/RandomFractals/vscode-data-table/assets/146131347/4a477bb4-bba3-4aaf-805d-e6f286534350)

Please consider adding:

```
.github-octo-flat-ui {
  line-height: 1.0 !important;
}
```

Result:
![image](https://github.com/RandomFractals/vscode-data-table/assets/146131347/80f7d21f-b2be-45e3-9620-9c365a9ad06c)

**Comment by RandomFractals** [comment]
@entact-vhorvath yes, I did not notice this when I reduced row height to display more rows. Will reduce the default font size in the next update to patch this.

**Comment by entact-vhorvath** [comment]
Perhaps it would be useful to give users the option to change this setting via the VSCode settings interface. Could be a few profiles that would allow users to pick from font sizes and packing of the rows (line heights), or a direct mapping between CSS values, whatever better fits the purpose. This extension might be combined with others and it is hard to inject a CSS rule from the outside, config seems to be a better way where you also can sanitize input.
